contracts/Token/OpThales_L2.sol

```diff
@@ -36,6 +43,18 @@ contract OpThales is IL2StandardERC20, ERC20 {
     ) public ERC20(_name, _symbol) {
         l1Token = _l1Token;
```

Comment:
whats the point of default values in lines 9 and 10 if it has to be passed through constructor?

Reply:
No point at all